lms/djangoapps/courseware/access_utils.py

```diff
@@ -61,17 +70,30 @@ def check_start_date(user, days_early_for_beta, start, course_key):
         AccessResponse: Either ACCESS_GRANTED or StartDateError.
     """
     start_dates_disabled = settings.FEATURES['DISABLE_START_DATES']
-    if start_dates_disabled and not is_masquerading_as_student(user, course_key):
+    masquerading_as_student = is_masquerading_as_student(user, course_key)
+    masquerading_as_specific_student = is_masquerading_as_specific_student(user, course_key)
+
+    if start_dates_disabled and not masquerading_as_student:
         return ACCESS_GRANTED
     else:
         now = datetime.now(UTC)
-        if start is None or in_preview_mode() or get_course_masquerade(user, course_key):
+        if start is None or in_preview_mode():
             return ACCESS_GRANTED
 
         effective_start = adjust_start_date(user, days_early_for_beta, start, course_key)
         if now > effective_start:
             return ACCESS_GRANTED
 
+        if get_course_masquerade(user, course_key):
```

Comment:
this is where the actual change for the ticket is happening.  
i'm continuing to grant access to content with a start date in the future, but this time I am logging a message.  
I added functionality to log the message only once per queue, since this code gets executed multiple times